### DISCUSSION TITLE
Buttons and top global menu panel  look.

### DISCUSSION THREAD

**Discussion by alphamodh0** [helloSystem/hello#314] [Ideas]
I think that is better a gradient like this:
![Botones](https://user-images.githubusercontent.com/90866187/148667153-27a62b4c-1802-4129-ba5d-28ee8aee69ea.png)

than this:
![Botoneshello](https://user-images.githubusercontent.com/90866187/148667159-2f7ead0c-2761-4aba-b8c0-22830d9b052c.png)

I love window manager theme and gradient:
![win](https://user-images.githubusercontent.com/90866187/148667178-5cf40cac-04a8-48f5-90a4-22783d4f6074.png)
I prefer that than this:
![menu](https://user-images.githubusercontent.com/90866187/148667194-8535c071-a525-4d4d-b43f-426c06cb6d3a.png)
I want to know what do yo think about.

**Comment by probonopd** [comment]
Undecided. 

![image](https://user-images.githubusercontent.com/2480569/148694414-78f405f4-7abf-458f-a5f4-dda949e27d30.png) = Aqua-ish (up to 10.4)

![image](https://user-images.githubusercontent.com/2480569/148694410-ea0525e8-2a42-4f13-ba38-df221d235fb8.png) = Leopard-ish (10.5+, increasingly used in later versions)

**Reply by probonopd** [reply]
You can play around in `/usr/local/etc/xdg/stylesheet.qss` to change these gradients as you like. Feel free to post screenshots if you think you have something that should be considered.

**Reply by alphamodh0** [reply]
![Screenshot_20220118-204753_DuckDuckGo.jpg](https://user-images.githubusercontent.com/90866187/150048299-f78590bc-7780-40a0-bbef-e21f79c64af3.jpg)
I think this kind of desing is better for me.

**Reply by alphamodh0** [reply]
> Always loved the way Aqua looked, but could never even dream about getting a mac back then, the good old days when icons & logos were still shiny and 3D xD 
> 
> So might be a tad bit biased when saying that Aqua would be the one that looks better to me 😅

I love shiny 3D and real-like icons, but do'nt like a poorly polished metal texture.
